app/models/book.rb

```diff
@@ -25,4 +27,6 @@ class Book < ApplicationRecord
     {less_than_or_equal_to: Settings.book.quantity.length, only_integer: true}
 
   delegate :title, to: :category, prefix: true, allow_nil: true
+
+  scope :get_cart, ->(array_ids){where id: array_ids}
```

Comment:
đặt lại tên scope

Reply:
```suggestion
  scope :by_book_ids, ->(ids){where id: ids}
```
e chú ý cách đặt tên nhé